setup.py

```diff
@@ -65,6 +65,7 @@
         "python-multipart",
         "requests",
         "sqlalchemy>=1.4",
-        "uvicorn",
+        "uvicorn < 0.17.0; python_version <= '3.6'",
+        "uvicorn >= 0.17.0; python_version > '3.6'",
```

Comment:
Do we explicitly need `uvicorn >= 0.17.0` when using Python 3.7 or higher, or should we rather specify just `uvicorn`?

Reply:
I took ert as an inspiration 🤷 
https://github.com/equinor/ert/blob/09152c9a300fe90a4b1fc046f10c6e8118c5ada5/setup.py#L106-L107